.github/workflows/ci_build.yml

```diff
@@ -11,7 +11,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.5, 3.6, 3.7, 3.8]
+        python-version: [3.5, 3.8]
+        pytorch-version: [0.4.1, 1.5.0]
```

Comment:
We can use `exclude` and `include` here to solve the problem mentioned in the comment.

```yaml
matrix:
  python-version: [3.5, 3.8]
  pytorch-version: [0.4.1, 1.5.0]
  exclude:
    - python-version: 3.8
      pytorch-version: 0.4.1
  include:
    - python-version: 3.7
      pytorch-version: 0.4.1
```

With this setting, jobs will be
```
ci-build (3.5, 0.4.1)
ci-build (3.5, 1.5.0)
ci-build (3.8, 1.5.0)
ci-build (3.7, 0.4.1)
```